app/views/content/_actions.html.erb

```diff
@@ -0,0 +1,46 @@
+<!-- draft mode -->
+<% if action_name.in? %w{edit layout annotate} %>
```

Comment:
I think this is a great case for some kind of presenter. It would allow you to replace things like `action_name.in? %w{edit layout annotate}` with `some_presenter.user_interacting?` or a better name.